src/release/breaking-changes/describe-enum.md

```diff
@@ -56,21 +70,28 @@ API documentation:
 
 * [`describeEnum` stable][]
 * [`describeEnum` main][]
+* [`EnumProperty` stable][]
+* [`EnumProperty` main][]
+* [enhanced enums]: https://github.com/dart-lang/language/blob/main/working/0158%20-%20Enhanced%20Enum/proposal.md
 
 Relevant issues:
 
-* [☂️ Cleanup SemanticsFlag and SemanticsAction issue][]
+* [cleanup-issue][]
 
 Relevant PRs:
 
 * [Deprecate `describeEnum` PR][]
 
 [`describeEnum` stable]: {{site.api}}/flutter/lib/src/foundation/describeEnum.html
 
+[`EnumProperty` stable]: {{site.api}}/flutter/lib/src/foundation/EnumProperty.html
+
 <!-- Master channel link: -->
 {% include docs/master-api.md %}
 
 [`describeEnum` main]: {{site.master-api}}/flutter/lib/src/foundation/describeEnum.html
 
-[☂️ Cleanup SemanticsFlag and SemanticsAction issue]: {{site.repo.flutter}}/issues/123346
```

Comment:
Why did you delete this line? It should be:

```
- [Cleanup SemanticsFlag and SemanticsAction issue][cleanup-issue]
```

And then [cleanup-issue] is defined below. It allows a shortcut to that loooong title.

Reply:
I don't understand but I think I did this now. I put the long title reference in the related issues, and in the master link it links to the shorter title.